qiskit/transpiler/passes/__init__.py

```diff
@@ -133,6 +133,7 @@
 from .optimization import RemoveResetInZeroState
 from .optimization import RemoveDiagonalGatesBeforeMeasure
 from .optimization import CrosstalkAdaptiveSchedule
+from .hoare_opt import HoareOptimizer
```

Comment:
this should go in the optimization directory

Reply:
done